src/dispatchers/keyEventDispatcher.ts

```diff
@@ -0,0 +1,67 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Dispatcher {
+  export class KeyEvent {
+    private static _DISPATCHER_KEY = "__Plottable_Dispatcher_Key";
+    private _connected = false;
+    private _keydownBroadcaster: Core.Broadcaster<Dispatcher.KeyEvent>;
+    // private _keyupBroadcaster: Core.Broadcaster<Dispatcher.KeyEvent>;
```

Comment:
I'll remove it for now. Minimal touch, just trying to replace the old dispatchers.